prow/plugins/releasenote/releasenote.go

```diff
@@ -114,6 +115,13 @@ func handleComment(gc githubClient, log *logrus.Entry, ic github.IssueCommentEve
 		return nil
 	}
 
+	// Emit deprecation warning for /release-note and /release-note-action-required.
+	if nl == releaseNote || nl == releaseNoteActionRequired {
+		format := "the `/%s` and `/%s` commands have been deprecated.\nPlease edit the `release-note` block in the PR body text to include the release note. If the release note requires additional action include the string `action required` in the release note. For example:\n````\n```release-note\nSome release note with action required.\n```\n````"
```

Comment:
Oh, so that's how you quote triple quotes!
````
```release-note
Some release note with action required.
```
````

Reply:
Yep. You can do it by bracketing with 2 backticks instead of 4 as well, but that version doesn't display multiple lines.